examples/PhasorDynamics/Example2/example2.cpp

```diff
@@ -85,17 +81,25 @@ int main()
   const index_type nt     = 2401;
   scalar_type      results[stride];
   buffer.seekg(0, std::ios::beg);
-  FILE* f = fopen("example2_results.csv", "w");
-  fprintf(f, "Time,gen2speed,gen3speed,v2mag,v3mag\n");
-  fprintf(f, "0,1,1,1,1\n");
+
+  std::ostream nullout(nullptr);
+  std::ostream& out = nullout;
+
+  // // Uncomment code below to print output to a file:
```

Comment:
Rather than uncommenting code this should probably be surrounded with `#IFDEF` so that you can switch with cmake.

Reply:
We will eventually add command line input for these things. This is temporary solution for now.